grails-app/controllers/ChartController.groovy

```diff
@@ -187,8 +187,17 @@ class ChartController {
         // We add the key to our cache set
         chartService.keyCache.add(concept)
 
+        // FIXME: get filters from concept data
```

Comment:
I now removed the comment, we approached it differently than in the data export screen (where we do combine filter and concept key in one object).